### DISCUSSION TITLE
How to use ReloadIndicator?

### DISCUSSION THREAD

**Discussion by martinzsir** [Netflix/dgs-framework#447] [General]
I want dynamic update schema based on the event.

How to use reloadIndicator?

**Comment by paulbakker** [comment]
This is the default implementation of the `ReloadIndicator`: https://github.com/Netflix/dgs-framework/blob/master/graphql-dgs-spring-boot-oss-autoconfigure/src/main/kotlin/com/netflix/graphql/dgs/autoconfig/DgsAutoConfiguration.kt#L118

As you can see in the code, it checks for either:

1. A configuration property called `dgs.reload` 
2. A profile `laptop`.

In your case, you probably want to override this behavior by registering a bean that implements the `ReloadIndicator` interface, which contains whatever logic you need to decide to set reload to true.
